### PR TITLE
Choose message post comments backoffice

### PR DESCRIPTION
**Pull request by BenZen59**

### CODE REVIEW

**Review by KeerhemFR — DISMISSED** [review]
See no evil

File: src/components/Comments/Post.jsx [hunk 1]
```diff
@@ -15,10 +16,14 @@ const CommentsPost = (props) => {
...
+        <ReferenceInput
+          label="Commentaire"
           source="idParent"
-          choices={[{ id: '1', name: 'Admin' }]}
-        />
+          reference="comments"
+        >
+          <SelectInput optionText="message" />
+        </ReferenceInput>
+        {/* <TextInput source="idParent" initialValue="1" /> */}
```
Comment (Hacene010) [inline src/components/Comments/Post.jsx]: dead code !

**Review by Hacene010 — CHANGES_REQUESTED** [review]

File: src/components/Comments/Post.jsx (hunk 1 above)
Comment (KeerhemFR) [reply src/components/Comments/Post.jsx]: oupsy

**Review by KeerhemFR — DISMISSED** [review]
See no more evil nor dead code

**Review by KeerhemFR — APPROVED** [review]
See no evil

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**